### DISCUSSION TITLE
[git] Push/Pull

### DISCUSSION THREAD

**Discussion by juyoungkimm** [Suanna01/HowIsTheWeatherToday2#16] [Q&A] [answered]
현재 작업하고 있는 branch는 feature_like인데 main에 올라와있는 코드를 풀 받으려면 branch만 변경하면 되나요?!
해당 사진처럼 나와서 질문 올립니다!
![KakaoTalk_Photo_2022-06-11-22-06-49](https://user-images.githubusercontent.com/80024989/173189286-9bddadc5-79e8-4159-a820-fe4ff1f1281e.jpeg)

**Comment by Suanna01** [comment, marked as answer]
현재 Branch에서 변경 사항이 있는 파일들을 모두 Commit&Push하고 Checkout 하면 됩니다!
강제 체크 아웃을 하면 현재 Branch에서 변경한 내용들이 모두 사라질 수 있으므로 주의해야 합니다!
